lib/state/state.js

```diff
@@ -375,14 +388,13 @@ function serializeEvent(path, data, isJson) {
 }
 
 
-State.prototype.broadcast = function (path, data, isJson) {
-	var evt = serializeEvent(path, data, isJson);
+State.prototype.broadcast = function (path, data, options) {
```

Comment:
I guess this is worthy of JSDoc :)

Reply:
TBH I now tend to put user doc only in the typedoc file. We might want to discuss this soon.